src/containerapp/azext_containerapp/tests/latest/test_containerapp_env_commands.py

```diff
@@ -460,6 +460,32 @@ def test_containerapp_env_internal_only_e2e(self, resource_group):
         ])
 
 
+    @ResourceGroupPreparer(location="eastus")
+    @live_only()  # passes live but hits CannotOverwriteExistingCassetteException when run from recording
```

Comment:
nit: do you think we should get this checked out? I know this is true for some test and not all test?

Reply:
this error is bc the request prepared to be sent (but actually won't) doesn't match the one on record from the recording. to debug, check the CannotOverwriteExistingCassetteException for the != line -- this will show you where the mismatch is happening and that way you can tune the test to have a reproducible outcome.